core/storage/storage.go

```diff
@@ -551,3 +551,16 @@ func DeleteStoredData(store Storage, metaData common.MetaData) common.SyncServic
 
 	return store.DeleteStoredData(metaData.DestOrgID, metaData.ObjectType, metaData.ObjectID)
 }
+
+// Only for CouchDB
+func createDSN(ipAddress, username, password string) string {
```

Comment:
I noticed that some 'create path' like functions were in storage.go, even if they were specific to a particular database. For example, [this](https://github.com/open-horizon/edge-sync-service/blob/cceaa2269651f163b6955bfbc6dba4b6ecf6f418/core/storage/storage.go#L408) and [this](https://github.com/open-horizon/edge-sync-service/blob/cceaa2269651f163b6955bfbc6dba4b6ecf6f418/core/storage/storage.go#L420) are specific to Bolt. This function was similar so I put it here, should I move it to couchStorage.go or couchHelpers.go?

Reply:
I didnt realize there were DB specific functions in there. Perhaps those functions were originally expected to be re-used but have since become bolt only. I think you should move createDSN to couchStorageHelpers.go.